src/server/graphql/mutations/addSlackChannel.js

```diff
@@ -3,9 +3,10 @@ import getRethink from 'server/database/rethinkDriver';
 import AddSlackChannelPayload from 'server/graphql/types/AddSlackChannelPayload';
 import getPubSub from 'server/utils/getPubSub';
 import {requireSUOrSelf, requireSUOrTeamMember, requireWebsocket} from 'server/utils/authorization';
-import shortid from 'shortid';
 import {SLACK} from 'universal/utils/constants';
+import insertSlackChannel from 'server/safeMutations/insertSlackChannel';
```

Comment:
this pattern came about during the organizations epic, but i didn't refactor them into their own folder.
sometimes, 2 mutations need to execute the same back-end code. For example, removeOrgMember and removeTeamMember will both execute the same shared code to remove a team member. However, that back-end helper function shouldn't live under the `TeamMember` folder, since it's really used equally by both.

 now that i'm housing relay-related calls in their own flat folder hierarchy (types, mutations, queries), i started putting their helpers in the `safeMutations` folder. Essentially, it is a helper function that runs some back-end stuff with no Auth or Validation (hence, the "safe" part). The idea is that whenever 2 or more mutations need to execute the same database operation, we can DRY it out by moving the shared code to a safeMutation.